### PR TITLE
Added VirtualBox ISO builder option to create additional disks

### PR DESCRIPTION
**Pull request by dreibh**
Added option to specify "disk_additional_size" for an array of sizes for additional disks, similar to the corresponding QEMU builder option.

### CODE REVIEW

**Review by sylviamoss — APPROVED** [review]
LGTM!

File: builder/virtualbox/iso/builder.go [hunk 1]
```diff
@@ -101,6 +101,12 @@ type Config struct {
 	// When set to sata, the drive is attached to an AHCI SATA controller.
 	// When set to virtio, the drive is attached to a VirtIO controller.
 	ISOInterface string `mapstructure:"iso_interface" required:"false"`
+	// Additional disks to create. Uses `vm_name` as the disk name template and
+	// appends `-#` where `#` is the position in the array. `#` starts at 1 since 0
+	// is the default disk. Each string represents the disk image size in MiB.
```
[the remark this answers is not in the document]
Comment (dreibh) [reply builder/virtualbox/iso/builder.go]: Updated.

File: builder/virtualbox/common/step_attach_isos.go [hunk 2]
```diff
@@ -69,49 +71,50 @@ func (s *StepAttachISOs) Run(ctx context.Context, state multistep.StateBag) mult
...
 			controllerName = "IDE Controller"
-			port = "1"
-			device = "0"
+			port = 1
+			device = 0
 			if s.GuestAdditionsInterface == "sata" {
 				controllerName = "SATA Controller"
-				port = "2"
-				device = "0"
+				port = 14
```
[the remark this answers is not in the document]
Comment (dreibh) [reply builder/virtualbox/common/step_attach_isos.go:98]: With additional disks, the ports 1, 2, ... may be used for HDDs. So, the ports for ISO and Guest Additions are numbered backwards.

**Comment by sylviamoss** [comment]
I pushed a commit to fix the generated docs error.

**Pull request merged**